container/container.go

```diff
@@ -232,6 +232,9 @@ func (container *Container) ExitOnNext() {
 // Resize changes the TTY of the process running inside the container
 // to the given height and width. The container must be running.
 func (container *Container) Resize(h, w int) error {
+	if container.Command.ProcessConfig.Terminal == nil {
+		return fmt.Errorf("Terminal is not ready")
```

Comment:
I think the message should be slightly more descriptive, something like:

```
"container %s does have a terminal ready", container.ID
```

or something like that